projects/2021_05_09_user_operated_internet_fund.md

```diff
@@ -508,6 +508,187 @@ Metrics regarding applications to the October call
+
+> Another call ("The February call") will open on December 1st with deadline February 1st.
+
+> After a report is delivered on this milestone and if it is approved by the NS, the NS will send the amount of PKT needed to raise the balance of the Trust's treasury to the Buffer Amount.
+
+
+We are submitting M3 on March 8th 2022. This includes the results from the 2021-10U (October 1st) and 2021-12U (December 1st), which were merged due to financial delays. The two rounds combined had 43 proposals.
+
+Six new projects were granted, bringing the total to 10 regular projects originating from 7 countries: Argentina (1 proposal, 9%), Germany (4 proposals, 36%), South Korea (1 proposal, 9%), Mexico (1 proposal, 9%), Slovenia (1 proposal, 9%), United States (2 proposals, 18%), South Africa (1 proposal, 9%). 
+
+The total amount requested so far was € 3 031 600, with applicants from 29 countries that together submitted 87 projects. 
+
+
+### 1. The balance of payments
+
+The balance of payments to the User Operated Internet Fund
+
+- 20mn PKT was donated by the Network Steward so far.
+
+- 20mn PKT was converted into € 377 240. See #4.
+
+The current balance is 0 PKT.
+
+### 2. Converted from PKT
+
+The amount of money which was converted from PKT during the reported period
+
+    10mn PKT -> 159740,16 (180.000 dollar) 
+
+### 3. Events of selling PKT
+
+These have been declared in status updates at the time of the conversion.
+
+### 4. Financial status
+
+An overview of the financial status of the fund, including a breakdown of costs (including administrative costs) and financial interaction with the Technology Commons Trust.
+
+As reported before, the launch of the User-Operated Internet Fund was significantly more successful than anticipated. We have temporarily frozen the tentative support project budget (max 30%, currently 2.7%) and similarly shifted a total of 22,162 € from the administrative budget (fixed 10%, now 3.7%) to the regular project budget to allow for the most beneficial projects to be granted from this call/M2. This included some eligible projects which were pushed to the October 2021 call from the previous round (see report M2). 
+
+Replenishment of funds to restore the administrative budget to the required 10% and to allocate budget to support projects is requested for the next round.
+
+* Assets as of March 8th 2022
+
+  * Total available assets: 0 PKT, 377,240€
+  * 325,802 € budget allocated to regular projects (93.8%)
+    * Spent: 28 200 €
+  *  10,000 € (of max 113,172 €) allocated to support projects (2.7%, max 30%)
+    * Spent: 0€
+  *  13,238 € administrative budget (fixed 10%, temporarily reduced to 3.7%)
+    * Spent: 20,163€
+    * Prefunded by NLnet; 6925€
```

Comment:
The currency format is non consistent in this section. Should all be €<amount>  to match the rest of the doc.